domain/ide/workspace/folder.go

```diff
@@ -610,6 +611,7 @@ func isVisibleSeverity(issue snyk.Issue) bool {
 func (f *Folder) publishDiagnostics(product product.Product, issuesByFile snyk.IssuesByFile) {
 	f.sendHovers(issuesByFile)
 	f.sendDiagnostics(issuesByFile)
+	ui.SendDiagnosticsOverview(f.c, product, issuesByFile, f.notifier)
 	f.sendSuccess(product)
```

Comment:
I think we need to rethink the flow here

Currently if scan success -> load issues from diagnostics.

In the future:
it doesn't matter if success or not IDE reacts to the new notification to load the HTML panel in the IDE

Maybe we don't need the f.sendSuccess anymore?

Reply:
Actually, yes and no. We will send this by scan & product. So actually it could be combined. Although I like that scan status reporting is separated from the actual overview html. I'm open for both, though.